ddtrace/settings/config.py

```diff
@@ -222,6 +222,8 @@ def __init__(self):
 
         self.health_metrics_enabled = asbool(os.getenv("DD_TRACE_HEALTH_METRICS_ENABLED", default=False))
 
+        self.telemetry_metrics_enabled = asbool(os.getenv("DD_TELEMETRY_METRICS_ENABLED", default=False))
```

Comment:
We can keep this setting internal. This way we can easily remove it if it’s rarely used. 

```suggestion
        self.telemetry_metrics_enabled = asbool(os.getenv("_DD_TELEMETRY_METRICS_ENABLED", default=False))
```